### DISCUSSION TITLE
Github actions logs contrast is too low in dark mode

### DISCUSSION THREAD

**Discussion by gpotter2** [community/community#22208] [Accessibility]
<p>The github actions logs appear as dark text on dark background when using dark mode. The contrast is not ideal.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/5/f518d1769afb63904bba4021f3540f9761bfc55b.png" data-download-href="/uploads/short-url/yYe9uilbyTjcgvS8wMUCHyJKrBp.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/10530980/181069514-d7221477-af60-4ae4-b404-f4f725a5b037.png" alt="image" data-base62-sha1="yYe9uilbyTjcgvS8wMUCHyJKrBp" width="690" height="378" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/5/f518d1769afb63904bba4021f3540f9761bfc55b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1384×759 53.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>Compare that to light mode:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/6/c/6cb8e2670fba3b82704333d4ec47d9b3fcc31989.png" data-download-href="/uploads/short-url/fvNG93P0gwjUW3RkOx1ffXa4xnj.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/10530980/181069516-0f4c385f-e1f7-439e-916d-7e4f681c1fef.png" alt="image" data-base62-sha1="fvNG93P0gwjUW3RkOx1ffXa4xnj" width="690" height="358" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/c/6cb8e2670fba3b82704333d4ec47d9b3fcc31989_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">840×436 18.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by gaknoll** [comment, marked as answer]
<p>Thanks for sharing the feedback <a class="mention" href="/gpotter2">@gpotter2</a>! We have received quite a bit of community feedback about the contrast so we expect to tweak views like this over the coming weeks</p>

**Comment by samollari** [comment]
<p>I couldn’t even see the log text with dark mode enabled… I had to disable it for a minute to see what my actions were doing</p>

**Comment by cehfisher** [comment]
Hi @i-am-gizm0 - I wanted to follow-up on this stale issue and ask if the dark mode theme changes have resolved this issue for you? If so, could you close this issue? If not, what additional changes are needed? (cc @queenofcorgis)

**Reply by samollari** [reply]
Hi, I wasn't the one to open the issue, but I haven't noticed a problem with it for a while so I think it's safe to say it's resolved! I believe @gpotter2 is the one who can close this discussion?

**Reply by gpotter2** [reply]
I believe it is already marked as resolved !

**Reply by cehfisher** [reply]
Thanks for confirming! I'm doing some clean-up of stale open issues. @queenofcorgis could you please close this issue? Thanks!

**Reply by gpotter2** [reply]
My bad. I never use github issues. TIL that Answered != Closed.

**Comment by Ahmedmeshref280** [comment]
> <p>The github actions logs appear as dark text on dark background when using dark mode. The contrast is not ideal.</p>
> <p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/5/f518d1769afb63904bba4021f3540f9761bfc55b.png" data-download-href="/uploads/short-url/yYe9uilbyTjcgvS8wMUCHyJKrBp.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/10530980/181069514-d7221477-af60-4ae4-b404-f4f725a5b037.png" alt="image" data-base62-sha1="yYe9uilbyTjcgvS8wMUCHyJKrBp" width="690" height="378" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/5/f518d1769afb63904bba4021f3540f9761bfc55b_2_10x10.png"><div class="meta">
> <svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1384×759 53.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
> </div></a></div>
> <p>Compare that to light mode:</p>
> <p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/6/c/6cb8e2670fba3b82704333d4ec47d9b3fcc31989.png" data-download-href="/uploads/short-url/fvNG93P0gwjUW3RkOx1ffXa4xnj.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/10530980/181069516-0f4c385f-e1f7-439e-916d-7e4f681c1fef.png" alt="image" data-base62-sha1="fvNG93P0gwjUW3RkOx1ffXa4xnj" width="690" height="358" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/c/6cb8e2670fba3b82704333d4ec47d9b3fcc31989_2_10x10.png"><div class="meta">
> <svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">840×436 18.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
> </div></a></div>

**Comment by Ahmedmeshref280** [comment]
> > <p>The github actions logs appear as dark text on dark background when using dark mode. The contrast is not ideal.</p>
> > <p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/5/f518d1769afb63904bba4021f3540f9761bfc55b.png" data-download-href="/uploads/short-url/yYe9uilbyTjcgvS8wMUCHyJKrBp.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/10530980/181069514-d7221477-af60-4ae4-b404-f4f725a5b037.png" alt="image" data-base62-sha1="yYe9uilbyTjcgvS8wMUCHyJKrBp" width="690" height="378" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/5/f518d1769afb63904bba4021f3540f9761bfc55b_2_10x10.png"><div class="meta">
> > <svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1384×759 53.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
> > </div></a></div>
> > <p>Compare that to light mode:</p>
> > <p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/6/c/6cb8e2670fba3b82704333d4ec47d9b3fcc31989.png" data-download-href="/uploads/short-url/fvNG93P0gwjUW3RkOx1ffXa4xnj.png?dl=1" title="image" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/10530980/181069516-0f4c385f-e1f7-439e-916d-7e4f681c1fef.png" alt="image" data-base62-sha1="fvNG93P0gwjUW3RkOx1ffXa4xnj" width="690" height="358" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/c/6cb8e2670fba3b82704333d4ec47d9b3fcc31989_2_10x10.png"><div class="meta">
> > <svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">840×436 18.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
> > </div></a></div>
> 
> 

- [ ]

**Reply by Ahmedmeshref280** [reply]
> https://github.com/community/community/discussion- s/22208#discussioncomment-1378070[]
